### PR TITLE
Update langversion-compiler-option.md

### LINKED ISSUE #12689: Links are confusing
This is one of the most odd pages I've ever been to on Docs. 

First, the superscript tags are odd.
image

A. It's strange to see footnotes that are more than just 1, 2, 3 or a, b, c. I can't think of anywhere else I've seen named footnotes.
B. All the links essentially go to the exact same spot on the page (since everything is close together), making them kind of useless.

Then, at the bottom, there are all these other odd links
image

When I first saw this, I couldn't even figure out what I was looking at. Specific issues:

1) The return arrow is an odd choice. I thought it was a typo. Is there a precedent for this?
2) Only the return arrow, links back up. The text portion of the links do nothing.
3) "CS80", for example, should probably be "C# 8.0"

In my opinion, every single one of these links (in both images) are completely useless and overkill.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9a3c8595-6bf5-61b0-fe12-f6cbbb1e2da8
* Version Independent ID: ac04cb00-0bfc-7bde-885d-c1dce0f35b4c
* Content: [-langversion (C# Compiler Options)](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/langversion-compiler-option#FCS71)
* Content Source: [docs/csharp/language-reference/compiler-options/langversion-compiler-option.md](https://github.com/dotnet/docs/blob/master/docs/csharp/language-reference/compiler-options/langversion-compiler-option.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by alx168**
## Summary

Removing superscripts to make the page less confusing to navigate.

Fixes #12689

### CODE REVIEW

**Comment by svick** [comment]
If the superscripts are removed, something should also be done with the section where they point to.

**Comment by BillWagner** [comment]
@alx168 Thanks for making this change.

I agree with @svick This PR should also remove the anchors from the list starting on line 78. Once that's done, I'll merge this PR.

**Comment by alx168** [comment]
Ok, thanks. @BillWagner should I create a new PR for that? I'm unsure of how to add a new commit to an existing PR since all I did was edit the docs on github.

**Comment by Youssef1313** [comment]
@alx168, There is no need to create a new PR. Just continue pushing commits in your branch and the changes will be reflected in the PR automatically.
[Here](https://github.com/alx168/docs/blob/patch-1/docs/csharp/language-reference/compiler-options/langversion-compiler-option.md) is the file in your branch.

**Comment by BillWagner** [comment]
@alx168 From the link provided in the previous comment, you can click the pencil icon to make more edits. That will update the PR.

**Comment by alx168** [comment]
Let me know if there's anything else you would recommend. Thanks a lot for your help @BillWagner @Youssef1313

File: docs/csharp/language-reference/compiler-options/langversion-compiler-option.md [hunk 1]
```diff
@@ -75,15 +75,14 @@ The default language version depends on the target framework for your applicatio
 |C# 1.0|[Download DOC](https://www.ecma-international.org/publications/files/ECMA-ST-ARCH/ECMA-334%201st%20edition%20December%202001.pdf)|C# Language Specification Version 1.0: Microsoft Corporation|
 
 ### Minimum compiler version needed to support all language features
```
Comment (Youssef1313) [inline docs/csharp/language-reference/compiler-options/langversion-compiler-option.md]:
```suggestion
### Minimum compiler version needed to support all language features

```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]

File: docs/csharp/language-reference/compiler-options/langversion-compiler-option.md [hunk 2]
```diff
@@ -29,17 +29,17 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
...
-|7.2|The compiler accepts only syntax that is included in C# 7.2 or lower <sup id="TCS72">[CS72](#FCS72)</sup>|
-|7.1|The compiler accepts only syntax that is included in C# 7.1 or lower <sup id="TCS71">[CS71](#FCS71)</sup>|
-|7|The compiler accepts only syntax that is included in C# 7.0 or lower <sup id="TCS7">[CS7](#FCS7)</sup>|
-|6|The compiler accepts only syntax that is included in C# 6.0 or lower <sup id="TCS6">[CS6](#FCS6)</sup>|
-|5|The compiler accepts only syntax that is included in C# 5.0 or lower <sup id="TCS5">[CS5](#FCS5)</sup>|
-|4|The compiler accepts only syntax that is included in C# 4.0 or lower <sup id="TCS4">[CS4](#FCS4)</sup>|
-|3|The compiler accepts only syntax that is included in C# 3.0 or lower <sup id="TCS3">[CS3](#FCS3)</sup>|
-|ISO-2|The compiler accepts only syntax that is included in ISO/IEC 23270:2006 C# (2.0) <sup id="TISO2">[ISO2](#FISO2)</sup>|
-|ISO-1|The compiler accepts only syntax that is included in ISO/IEC 23270:2003 C# (1.0/1.2) <sup id="TISO1">[ISO1](#FISO1)</sup>|  
+|8.0|The compiler accepts only syntax that is included in C# 8.0 or lower
```
Comment (Youssef1313) [inline docs/csharp/language-reference/compiler-options/langversion-compiler-option.md]:
```suggestion
|8.0|The compiler accepts only syntax that is included in C# 8.0 or lower.|
```

File: docs/csharp/language-reference/compiler-options/langversion-compiler-option.md [hunk 3]
```diff
@@ -29,17 +29,17 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
...
-|7.1|The compiler accepts only syntax that is included in C# 7.1 or lower <sup id="TCS71">[CS71](#FCS71)</sup>|
-|7|The compiler accepts only syntax that is included in C# 7.0 or lower <sup id="TCS7">[CS7](#FCS7)</sup>|
-|6|The compiler accepts only syntax that is included in C# 6.0 or lower <sup id="TCS6">[CS6](#FCS6)</sup>|
-|5|The compiler accepts only syntax that is included in C# 5.0 or lower <sup id="TCS5">[CS5](#FCS5)</sup>|
-|4|The compiler accepts only syntax that is included in C# 4.0 or lower <sup id="TCS4">[CS4](#FCS4)</sup>|
-|3|The compiler accepts only syntax that is included in C# 3.0 or lower <sup id="TCS3">[CS3](#FCS3)</sup>|
-|ISO-2|The compiler accepts only syntax that is included in ISO/IEC 23270:2006 C# (2.0) <sup id="TISO2">[ISO2](#FISO2)</sup>|
-|ISO-1|The compiler accepts only syntax that is included in ISO/IEC 23270:2003 C# (1.0/1.2) <sup id="TISO1">[ISO1](#FISO1)</sup>|  
+|8.0|The compiler accepts only syntax that is included in C# 8.0 or lower 
+|7.3|The compiler accepts only syntax that is included in C# 7.3 or lower
```
Comment (Youssef1313) [inline docs/csharp/language-reference/compiler-options/langversion-compiler-option.md]:
```suggestion
|7.3|The compiler accepts only syntax that is included in C# 7.3 or lower.|
```

File: docs/csharp/language-reference/compiler-options/langversion-compiler-option.md [hunk 4]
```diff
@@ -29,17 +29,17 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
...
+|7.2|The compiler accepts only syntax that is included in C# 7.2 or lower
```
Comment (Youssef1313) [inline docs/csharp/language-reference/compiler-options/langversion-compiler-option.md]:
```suggestion
|7.2|The compiler accepts only syntax that is included in C# 7.2 or lower.|
```

File: docs/csharp/language-reference/compiler-options/langversion-compiler-option.md [hunk 5]
```diff
@@ -29,17 +29,17 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
...
+|7.1|The compiler accepts only syntax that is included in C# 7.1 or lower
```
Comment (Youssef1313) [inline docs/csharp/language-reference/compiler-options/langversion-compiler-option.md]:
```suggestion
|7.1|The compiler accepts only syntax that is included in C# 7.1 or lower.|
```

File: docs/csharp/language-reference/compiler-options/langversion-compiler-option.md [hunk 6]
```diff
@@ -29,17 +29,17 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
...
+|7|The compiler accepts only syntax that is included in C# 7.0 or lower
```
Comment (Youssef1313) [inline docs/csharp/language-reference/compiler-options/langversion-compiler-option.md]:
```suggestion
|7|The compiler accepts only syntax that is included in C# 7.0 or lower.|
```

File: docs/csharp/language-reference/compiler-options/langversion-compiler-option.md [hunk 7]
```diff
@@ -29,17 +29,17 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
...
+|6|The compiler accepts only syntax that is included in C# 6.0 or lower
```
Comment (Youssef1313) [inline docs/csharp/language-reference/compiler-options/langversion-compiler-option.md]:
```suggestion
|6|The compiler accepts only syntax that is included in C# 6.0 or lower.|
```

File: docs/csharp/language-reference/compiler-options/langversion-compiler-option.md [hunk 8]
```diff
@@ -29,17 +29,17 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
...
+|5|The compiler accepts only syntax that is included in C# 5.0 or lower
```
Comment (Youssef1313) [inline docs/csharp/language-reference/compiler-options/langversion-compiler-option.md]:
```suggestion
|5|The compiler accepts only syntax that is included in C# 5.0 or lower.|
```

File: docs/csharp/language-reference/compiler-options/langversion-compiler-option.md [hunk 9]
```diff
@@ -29,17 +29,17 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
...
+|4|The compiler accepts only syntax that is included in C# 4.0 or lower
```
Comment (Youssef1313) [inline docs/csharp/language-reference/compiler-options/langversion-compiler-option.md]:
```suggestion
|4|The compiler accepts only syntax that is included in C# 4.0 or lower.|
```

File: docs/csharp/language-reference/compiler-options/langversion-compiler-option.md [hunk 10]
```diff
@@ -29,17 +29,17 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
...
+|3|The compiler accepts only syntax that is included in C# 3.0 or lower
```
Comment (Youssef1313) [inline docs/csharp/language-reference/compiler-options/langversion-compiler-option.md]:
```suggestion
|3|The compiler accepts only syntax that is included in C# 3.0 or lower.|
```

File: docs/csharp/language-reference/compiler-options/langversion-compiler-option.md [hunk 11]
```diff
@@ -29,17 +29,17 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
...
+|ISO-2|The compiler accepts only syntax that is included in ISO/IEC 23270:2006 C# (2.0)
```
Comment (Youssef1313) [inline docs/csharp/language-reference/compiler-options/langversion-compiler-option.md]:
```suggestion
|ISO-2|The compiler accepts only syntax that is included in ISO/IEC 23270:2006 C# (2.0).|
```

File: docs/csharp/language-reference/compiler-options/langversion-compiler-option.md [hunk 12]
```diff
@@ -29,17 +29,17 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
...
+|ISO-1|The compiler accepts only syntax that is included in ISO/IEC 23270:2003 C# (1.0/1.2)
```
Comment (Youssef1313) [inline docs/csharp/language-reference/compiler-options/langversion-compiler-option.md]:
```suggestion
|ISO-1|The compiler accepts only syntax that is included in ISO/IEC 23270:2003 C# (1.0/1.2).|
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]

**Review by BillWagner — APPROVED** [review]
thanks for helping us updated this article @alx168 

After incorporating the feedback from reviews, this is ready to :shipit:

Thanks for making your first contribution to docs! 🎆

**Pull request merged**